### PR TITLE
add a tutorial of ycsb bench in TiUP

### PR DESCRIPTION
**Pull request by Smityz**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-bench.md [hunk 1]
```diff
@@ -180,3 +180,23 @@ Flags:
...
     ```
+
+## 使用 YCSB 测试 TiKV
+
+1. 准备数据：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    # 必须指定 -c 参数，意义为插入数据条数
```
Comment (shichun-0415) [inline tiup/tiup-bench.md:191]: 条数是否=行数？
```suggestion
    # 必须指定 -c 参数，即插入数据的条数
```

File: tiup/tiup-bench.md [hunk 2]
```diff
@@ -180,3 +180,23 @@ Flags:
...
+    tiup bench ycsb prepare -c 10000
+    ```
+
+2. 运行 YCSB 测试：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    # 必须指定 -c 参数，意义为操作次数，默认 READ 流量占 95%, UPDATE 占 5%
```
Comment (shichun-0415) [inline tiup/tiup-bench.md:200]:
```suggestion
    # 必须指定 -c 参数，即操作次数，默认 READ 流量占 95%，UPDATE 占 5%
```

**Review by dbsid — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**